event-handlers/src/test/java/no/sikt/nva/nvi/events/evaluator/EvaluateNviCandidateWithSyntheticDataTest.java

```diff
@@ -216,6 +210,53 @@ void shouldThrowParsingExceptionWhenChannelIsMalformed() {
         .contains("Required field 'scientificValue' is null");
   }
 
+  @ParameterizedTest
+  @ValueSource(strings = {"AcademicChapter", "AcademicMonograph", "AcademicCommentary"})
+  void shouldEvaluateAsNonCandidateWhenPublicationRequiringIsbnIsWithoutIsbn(
+      String publicationType) {
+    var publication =
+        factory
+            .withPublicationType(publicationType)
+            .withContributor(verifiedCreatorFrom(nviOrganization))
+            .withPublicationChannel(PUBLISHER_TYPE, LEVEL_ONE)
+            .withIsbnList(Collections.emptyList())
+            .getExpandedPublication();
+
+    var candidate = getEvaluationResult(publication);
+    assertThat(candidate).isInstanceOf(UpsertNonNviCandidateRequest.class);
+  }
+
+  @ParameterizedTest
+  @ValueSource(strings = {"AcademicChapter", "AcademicMonograph", "AcademicCommentary"})
+  void shouldEvaluateAsCandidateResourcesRequiringIsbnWithIsbn(String publicationType) {
+    var publication =
+        factory
+            .withPublicationType(publicationType)
+            .withContributor(verifiedCreatorFrom(nviOrganization))
+            .withPublicationChannel(PUBLISHER_TYPE, LEVEL_ONE)
+            .withIsbnList(List.of(randomString()))
```

Comment:
nitpick: Doesn't actually matter, but we do have a `randomIsbn()` method for this.